code/modules/mob/mob.dm

```diff
@@ -1046,7 +1046,7 @@ var/list/slot_equipment_priority = list( \
 
 		src.pulling = P
 		P.pulledby = src
-		P.on_pull_start(AM)
+		AM.on_pull_start(P)
```

Comment:
This is actually how it was intended to be in the first place. I implemented this when adding cockatrices so that you'd get petrified when attempting to pull one. But because of this switchup that functionality didn't work